sympy/stats/joint_rv_types.py

```diff
@@ -188,11 +192,25 @@ def MultivariateT(syms, mu, sigma, v):
     Parameters
     ==========
 
-    syms: list/tuple/set of symbols for identifying each component
-    mu: A list/tuple/set consisting of k means,represents a k
-        dimensional location vector
+    syms: A symbol/str
+        For identifying the random variable.
+    mu: A list/matrix
+        Representing the location vector
     sigma: The shape matrix for the distribution
 
+    Examples
+    ========
+
+    >>> from sympy.stats import density
+    >>> from sympy.stats.joint_rv_types import MultivariateT
```

Comment:
IMO we can import in a single line as,
`>>> from sympy.stats import density, MultivariateT`

Reply:
@Smit-create Sure, I think this might be an easy way to import. I will make the changes. Thanks for the review!